examples/bri-3/zeroKnowledgeKeys/scripts/ptau.sh

```diff
@@ -8,38 +8,38 @@ set -e
 # make ptau folder
 [ -d zeroKnowledgeKeys/ptau ] || mkdir zeroKnowledgeKeys/ptau
 
-# Starts Powers Of Tau ceremony, creating the file pot14_0000.ptau
-# 14 is the power of two of the maximum number of constraints that the ceremony can accept: in this case, the number of constraints is 2 ^ 14 = 16,384.
-# 14 is required as ecdsaSignatureVerification takes 8,076 constraints
-snarkjs powersoftau new bn128 14 zeroKnowledgeKeys/ptau/pot14_0000.ptau -v
+# Starts Powers Of Tau ceremony, creating the file pot20_0000.ptau
```

Comment:
could we just use a readily available ptau file from the web for the sample use-case or we need to go through the ceremony?

Reply:
if groth16 is used then you need to run a trusted set-up for every invocation of a circuit with public and private inputs. That is not necessary with PLONK. There we can use an available common reference string (CRS) from the public power of tau ceremony that has already been done. So you see it depends what commitment schemes are used.

Reply:
We are using Groth16 for our circuit.